vllm/attention/backends/flashinfer.py

```diff
@@ -99,6 +104,71 @@ def get_fp8_dtype_for_flashinfer(kv_cache_dtype: str) -> torch.dtype:
             raise ValueError(f"Unrecognized FP8 dtype: {kv_cache_dtype}")
 
 
+@dataclass
+class GlobalHyperparameters:
+    '''
+    Currently, FlashInfer backend only support models in which all layers share
+    the same values for the following hyperparameters.
+    '''
+    window_left: int
+    logits_soft_cap: Optional[float]
+    sm_scale: float
+
+
+def infer_global_hyperparameters(model: nn.Module) -> GlobalHyperparameters:
```

Comment:
this function can collect all `per_layer_parameter`, and only assert the results are the same.